samza-core/src/test/java/org/apache/samza/zk/TestZkUtils.java

```diff
@@ -67,10 +68,11 @@ public void testSetup() {
       // Do nothing
     }
 
+    ZkJobCoordinatorMetrics metrics = new ZkJobCoordinatorMetrics(new NoOpMetricsRegistry());
```

Comment:
nit: You can avoid a local variable `metrics` by directly using new NoOpMetricsRegistry in the ZkUtils constructor.